.github/workflows/test.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v3
         with:
-          python-version: 3.x
+          python-version: "3.10"
```

Comment:
Are the quotes needed? If not please remove them. Otherwise I may want to add a comment to why we pin to 3.10 and add a bug reference for bikeshed?

Reply:
Yes the quotes are needed, otherwise YAML interprets it as the floating point number `3.1`…

I added a comment.